auditlog_async/README.rst

```diff
@@ -0,0 +1,43 @@
+Track user operation on data models (asynchronous mode)
+=======================================================
+
+This module overrides the ``auditlog`` module to make asynchronous the
+log creation, resulting in greater performance and a better user experience.
+
+Usage
+=====
+
+This module uses the ``connector`` framework to delay the log creation in a
+job queue.
+
+To take advantage of this module, you have to:
+
+ * run `Odoo` in multiprocess mode (``workers`` > 0)
+ * start `Odoo` with the ``ODOO_CONNECTOR_CHANNELS`` environment variable
+   defined and ``--load=web,connector``
```

Comment:
Perhaps add a link the connector documentation: http://odoo-connector.com/guides/jobrunner.html